### PR TITLE
Updates for Aarhus University

### PR DESCRIPTION
**Pull request by amoeller**

### CODE REVIEW

File: csrankings-i.csv [hunk 1]
```diff
@@ -265,7 +265,7 @@ Iordanis Koutsopoulos,AUEB,http://www.cs.aueb.gr/~jordan,Bq_GqyIAAAAJ
 Iovka Boneva,CRIStAL,https://pro.univ-lille.fr/iovka-boneva,ZxXztaAAAAAJ
 Ipek Oguz,Vanderbilt University,https://engineering.vanderbilt.edu/bio/ipek-oguz,oHJIjVwAAAAJ
 Iqbal Gondal,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/g/gondal---iqbal,BqlvfY4AAAAJ
-Ira Assent,Aarhus University,http://www.cs.au.dk/~ira,w2n5LhUAAAAJ
+Ira Assent,Aarhus University,https://cs.au.dk/~ira,w2n5LhUAAAAJ
```
Comment (emeryberger) [inline csrankings-i.csv]: Web page has no content.

File: csrankings-j.csv [hunk 2]
```diff
@@ -808,7 +809,7 @@ Jerzy W. Grzymala-Busse,University of Kansas,http://people.eecs.ku.edu/~jerzy,NO
 Jerzy W. Jaromczyk,University of Kentucky,http://www.cs.uky.edu/~jurek,NOSCHOLARPAGE
 Jes Frellsen,DTU,https://frellsen.org,Yj2sBWkAAAAJ
 Jesper Bengtson,IT University of Copenhagen,http://www.itu.dk/people/jebe,NOSCHOLARPAGE
-Jesper Buus Nielsen,Aarhus University,http://www.cs.au.dk/~jbn,PoDj4lsAAAAJ
+Jesper Buus Nielsen,Aarhus University,https://cs.au.dk/~jbn,PoDj4lsAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: Web page has no content.

File: csrankings-j.csv [hunk 3]
```diff
@@ -1261,7 +1262,6 @@ Jizhong Zhao,Xi'an Jiaotong University,http://www.xjtu.edu.cn/jsnr.jsp?urltype=t
 Jo Iacovides,University of York,https://www-users.york.ac.uk/~ii560,LuBaasYAAAAJ
 Jo Ueyama,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/joueyama,OXlxfi4AAAAJ
 Jó Ueyama,USP-ICMC,http://conteudo.icmc.usp.br/pessoas/joueyama,OXlxfi4AAAAJ
-Jo Vermeulen,Aarhus University,http://www.jovermeulen.com,mLqdIdoAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv:1264]: Please move to `old/industry.csv`.

File: csrankings-j.csv [hunk 4]
```diff
@@ -1747,7 +1747,6 @@ Jörg Henkel,Karlsruhe Institute of Technology,http://ces.itec.kit.edu/21_henkel
 Jörg Hoffmann 0001,Saarland University,http://fai.cs.uni-saarland.de/hoffmann,NR6qg_UAAAAJ
 Jörg Kienzle,McGill University,http://www.cs.mcgill.ca/~joerg/Home/Jorgs_Home.html,INUL3eEAAAAJ
 Jörg Liebeherr,University of Toronto,http://www.comm.utoronto.ca/~jorg,8WrYVG8AAAAJ
-Jörg Müller,Aarhus University,http://pure.au.dk/portal/en/persons/id(3e718f2c-7cc4-4cc5-8f05-87e65963d7c1).html,qU5PVzAAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv:1750]: Please restore but update affiliation and web page to University of Bayreuth, http://joergmueller.info/.

File: csrankings-l.csv [hunk 5]
```diff
@@ -68,7 +68,6 @@ Larry Latour,University of Maine,https://umaine.edu/cs,NOSCHOLARPAGE
 Larry S. Davis,University of Maryland - College Park,http://www.umiacs.umd.edu/~lsd,lc0ARagAAAAJ
 Larry Stillman,Monash University,http://monash.edu/research/explore/en/persons/larry-stillman(0e852ee0-bced-4b17-ba84-8b5defb30293).html,2J5mNZ0AAAAJ
 Larry Wasserman,Carnegie Mellon University,http://www.stat.cmu.edu/~larry,XcD1ffwAAAAJ
-Lars Arge,Aarhus University,http://cs.au.dk/~large,MgdxxbAAAAAJ
```
Comment (emeryberger) [inline csrankings-l.csv:71]: Please move to `old/rip.csv`.

File: csrankings-m.csv [hunk 6]
```diff
@@ -1569,7 +1569,6 @@ Michael Hübner,Brandenburg University of Technology,https://www.b-tu.de/en/fg-t
 Michael Huth,Imperial College London,http://www.doc.ic.ac.uk/~mrh,ZBlUaIAAAAAJ
 Michael I. Jordan,Univ. of California - Berkeley,http://www.cs.berkeley.edu/~jordan,yxUduqMAAAAJ
 Michael I. Mandel,CUNY,http://www.brooklyn.cuny.edu/web/academics/faculty/faculty_profile.jsp?faculty=1264,0v1-axgAAAAJ
-Michael I. Schwartzbach,Aarhus University,http://cs.au.dk/~mis,zX-HBWwAAAAJ
```
Comment (emeryberger) [inline csrankings-m.csv:1572]: Still listed here: https://cs.au.dk/~amoeller/mis/. Please clarify.

File: csrankings-o.csv [hunk 7]
```diff
@@ -102,7 +102,6 @@ Oliver Yang,University of Ottawa,http://www.site.uottawa.ca/~yang,NOSCHOLARPAGE
 Oliver van Kaick,Carleton University,http://people.scs.carleton.ca/~olivervankaick,IfvyBBUAAAAJ
 Olivier Alata,Université Jean Monnet,http://perso.univ-st-etienne.fr/ao29170h,ClqgTp8AAAAJ
 Olivier Caron,CRIStAL,https://www.cristal.univ-lille.fr/profil/carono,NOSCHOLARPAGE
-Olivier Danvy,Aarhus University,http://cs.au.dk/~danvy,OZJGoVoAAAAJ
```
Comment (emeryberger) [inline csrankings-o.csv:105]: Please update affiliation and web page (https://www.yale-nus.edu.sg/faculty/olivier-danvy/).

File: csrankings-s.csv [hunk 8]
```diff
@@ -1389,6 +1390,7 @@ Srijan Kumar,Georgia Institute of Technology,https://www.cse.gatech.edu/people/s
 Srikant Srinivasan,IIT Mandi,http://faculty.iitmandi.ac.in/~srikant_srinivasan,NOSCHOLARPAGE
 Srikanta Bedathur,IIT Delhi,http://www.cse.iitd.ac.in/~bedathur,e7Qs3JgAAAAJ
 Srikanta J. Bedathur,IIT Delhi,http://www.cse.iitd.ac.in/~bedathur,e7Qs3JgAAAAJ
+Srikanth Srinivasan,Aarhus University,https://cs.au.dk/~srikanth,uJzmr8YAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Needs 0001 disambiguation suffix (see https://dblp.org/pid/05/6302.html).

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Bump @amoeller

File: csrankings-o.csv [hunk 9]
```diff
@@ -102,6 +102,7 @@ Oliver Yang,University of Ottawa,http://www.site.uottawa.ca/~yang,NOSCHOLARPAGE
 Oliver van Kaick,Carleton University,http://people.scs.carleton.ca/~olivervankaick,IfvyBBUAAAAJ
 Olivier Alata,Université Jean Monnet,http://perso.univ-st-etienne.fr/ao29170h,ClqgTp8AAAAJ
 Olivier Caron,CRIStAL,https://www.cristal.univ-lille.fr/profil/carono,NOSCHOLARPAGE
+Olivier Danvy,Ysle NUS,https://www.yale-nus.edu.sg/faculty/olivier-danvy/,OZJGoVoAAAAJ
```
Comment (emeryberger) [inline csrankings-o.csv]: This institution (which incidentally is misspelled) is not in the database - faculty should be listed under NUS.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-j.csv (hunk 4 above)
Comment (emeryberger) [reply csrankings-j.csv:1750]: This still has not been done, @amoeller .

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-j.csv (hunk 4 above)
Comment (amoeller) [reply csrankings-j.csv:1750]: He is already there (as Jörg Müller 0001).

**Review by emeryberger — APPROVED** [review]

**Pull request merged**